### DISCUSSION TITLE
Request for assistance installation in laravel 10

### DISCUSSION THREAD

**Discussion by piyushpathar** [jenssegers/laravel-mongodb#2620] [Q&A] [answered]
I am writing to seek assistance with an installation issue I am encountering while trying to set up in Laravel 10. Specifically, I am facing a problem related to package compatibility. The error message I am receiving is as follow

Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - Root composer.json requires mongodb/laravel-mongodb ^3.9 -> satisfiable by mongodb/laravel-mongodb[v3.9.0, ..., v3.9.5].
    - mongodb/laravel-mongodb[v3.9.0, ..., v3.9.5] require illuminate/support ^9.0 -> found illuminate/support[v9.0.0, ..., v9.52.15] but it conflicts with your root composer.json require (^10.23).

**Comment by GromNaN** [comment, marked as answer]
For Laravel 10, the latest version is a release candidate [4.0.0-rc1](https://github.com/mongodb/laravel-mongodb/releases/tag/4.0.0-rc1). 

You can install it like this:

    composer require mongodb/laravel-mongodb:4.0.0-rc1

**Reply by piyushpathar** [reply]
Done , Thanks
